src/components/generics/vacation-test-forms/myVacationRequests.tsx

```diff
@@ -11,43 +11,38 @@ import Holidays from "date-holidays";
 import strings from "localization/strings";
 import { FilterScopes } from "types";
 import DateRangePicker from "../date-range-picker/date-range-picker";
-
-/**
- * interface type request
- */
-interface Request {
-  id: number;
-  vacationType: string;
-  comment: string;
-  employee: string;
-  days: number;
-  startDate: string;
-  endDate: string;
-  status: string;
-}
+import { Request } from "types/index";
+import { VacationRequest, VacationType } from "generated/client";
+import { useAppSelector } from "app/hooks";
+import { selectPerson } from "features/person/person-slice";
+import Api from "api/api";
+import { selectAuth } from "features/auth/auth-slice";
+import { ErrorContext } from "components/error-handler/error-handler";
 
 /**
  * Renders vacation request table
  */
 const renderVacationRequests = () => {
   const classes = useEditorContentStyles();
+  const { person } = useAppSelector(selectPerson);
+  const { accessToken } = useAppSelector(selectAuth);
   const [ dateFormat ] = useState("yyyy.MM.dd");
```

Comment:
As date format is not changed, this does not need to be state, rather it can just be a regular variable e.g.
const dateFormat = "yyyy.MM.dd".
Although as this is only used in one place it could just be passed directly into the DateRangePicker dateFormat prop.